salt/elife-xpub/init.sls

```diff
@@ -21,6 +21,16 @@ elife-xpub-repository:
         - require:
             - builder: elife-xpub-repository
 
+elife-xpub-logs:
+    file.directory:
+        - name: /srv/elife-xpub/var/logs/
+        - user: {{ pillar.elife.deploy_user.username }}
```

Comment:
I think the container at the moment runs as `root` but sooner or later a normal user should be substituted for that. At the moment this is in tune with the rest of the folder so this value (`elife`) is fine.